### PR TITLE
ECL-407 - Content iterations for private Beta - submit return journey

### PR DESCRIPTION
**Pull request by nemanjatomanovic**
- Added **privateBetaEnabled** feature flag
- Extended `ReturnSubmittedEmailParameters` with **privateBetaEnabled** so it can be used in email-renderer

### CODE REVIEW

File: conf/application.conf [hunk 1]
```diff
@@ -136,6 +136,9 @@ tracking-consent-frontend {
 features {
   welsh-translation = false
   enrolmentStoreProxyStubEnabled = true
+  privateBeta {
+      enabled = true
+    }
```
Comment (peter-hazell) [inline conf/application.conf:141]: very minor but formatting is slightly out here

**Comment by peter-hazell** [comment]
I think this is missing the change to hide the payments tile?

File: conf/application.conf (hunk 1 above)
Comment (nemanjatomanovic) [reply conf/application.conf]: Ah yes 😄 . Fixed now.

**Comment by nemanjatomanovic** [comment]
That is true @peter-hazell that PR will be on account-frontend service. I have a question regarding that change and as soon as it's resolved I will open in there.

**Comment by peter-hazell** [comment]
> That is true @peter-hazell that PR will be on account-frontend service. I have a question regarding that change and as soon as it's resolved I will open in there.

ah yes - of course sorry

**Comment by peter-hazell** [comment]
I think this is ready to approve, just need to update the `AlreadySubmittedReturn` view as discussed on slack to use the feature toggle

**Comment by nemanjatomanovic** [comment]
> I think this is ready to approve, just need to update the `AlreadySubmittedReturn` view as discussed on slack to use the feature toggle

Should we wait for DC PR first or we can go with this one ?

**Comment by peter-hazell** [comment]
Good point @nemanjatomanovic - wait for DC first otherwise we'll get email errors in QA

**Review by TonyCOgilvie — DISMISSED** [review]

**Review by TonyCOgilvie — APPROVED** [review]

**Pull request closed without merging**